minecraft-server/Dockerfile

```diff
@@ -43,6 +43,18 @@ RUN tar -x -C /usr/local/bin -f /tmp/rcon-cli.tgz rcon-cli && \
 ADD https://github.com/itzg/mc-server-runner/releases/download/${MC_SERVER_RUNNER_VER}/mc-server-runner_${MC_SERVER_RUNNER_VER}_linux_${ARCH}.tar.gz /tmp/mc-server-runner.tgz
 RUN tar -x -C /usr/local/bin -f /tmp/mc-server-runner.tgz mc-server-runner && \
   rm /tmp/mc-server-runner.tgz
+  
+ADD https://git.faldoria.de/tof/server/build-tools/-/jobs/artifacts/master/raw/target/ToF-BuildTools.jar?job=build /tmp/tof-buildtools/BuildTools.jar
+  
+ONBUILD ARG BUILDTOOLS_OUTPUT=/plugins
+ONBUILD COPY Dockerfile *plugins.yml /tmp/tof-buildtools/
```

Comment:
The Dockerfile must be copied over to avoid errors if no plugin.yml or *plugin.yml exists. Docker can not handle error cases and always expects a file to be copied when calling COPY.
By copying the Dockerfile the step always succeeds even if there are no specific plugin.yml files.

Reply:
In that case, nice solution :)